### PR TITLE
tests: enable allegedly flaky ingress cache test

### PR DESCRIPTION
**Pull request by kyessenov**
Signed-off-by: Kuat Yessenov <kuat@google.com>

### CODE REVIEW

**Comment by kyessenov** [comment]
@hklai per our discussion, let's enable it back. It passed from the first try, and I failed to reproduce it after repeatedly running it.

File: tests/e2e/tests/mixer/mixer_test.go [hunk 1]
```diff
@@ -729,7 +729,7 @@ func testDenials(t *testing.T, rule string) {
 
 // TestIngressCheckCache tests that check cache works in Ingress.
 func TestIngressCheckCache(t *testing.T) {
-	t.Skip("https://github.com/istio/istio/issues/6309")
+	//t.Skip("https://github.com/istio/istio/issues/6309")
```
Comment (hklai) [inline tests/e2e/tests/mixer/mixer_test.go]: Just remove the whole line, assuming this test is going to stay.

**Comment by kyessenov** [comment]
There is another skipped test there. I'll do the same thing to the other
test in a few days, and I can clean up the comment then.

**Pull request merged**